Examples/form-handler/click-element.robot

```diff
@@ -15,7 +15,11 @@ Click link
 Click Button
     Open browser to test page    http://127.0.0.1:7272/basic-html-elements.html
     Click Button    id=get_ajax
-    
+
+Double Click Element
+    Open browser to test page    http://127.0.0.1:7272/basic-html-elements.html
+    Double Click Element    id=double_click_get_id
```

Comment:
Could you and verify step for ensure that double click work as expect?
Due to some of library may not send mouse event as double click

Reply:
I suppose it's great to add it as Alias Keyword